pytorch_lightning/trainer/properties.py

```diff
@@ -477,6 +480,19 @@ def sanity_checking(self, val: bool) -> None:
     Loop properties
     """
 
+    @property
+    def epoch_loop(self) -> EpochLoop:
+        # TODO: the current train_loop should be renamed to epoch_loop
+        return self.train_loop
```

Comment:
maybe make the attribute protected and deprecate the self.train_loop property? May be confusing to get the difference between train_loop and training_loop